isis/src/base/objs/Cube/Cube.cpp

```diff
@@ -98,6 +98,10 @@ namespace Isis {
       putGroup(*grpIt);
     }
 
+    for (int i = 1; i < label.objects(); i++) {
```

Comment:
I understand why this is needed for this app. but this is concerning: At this point why don't we just copy the whole input PVL? Where else is this function used? What other tests will this influence?

Reply:
I thought the same, but it seems that things from the IsisCube PVL Object need to be set in a particular way? Almost like the init process creates a simple IsisCube Pvl Object so replacing it directly seems unsafe.